opentelemetry-sdk/src/export/log/mod.rs

```diff
@@ -0,0 +1,33 @@
+//! Log exporters
+use crate::{log::LogRecord, Resource};
+use async_trait::async_trait;
+use opentelemetry_api::log::LogError;
+use opentelemetry_api::{log::LogResult, InstrumentationLibrary};
+use std::{fmt::Debug, sync::Arc};
+
+pub mod stdout;
+
+/// `LogExporter` defines the interface that log exporters should implement.
+#[async_trait]
+pub trait LogExporter: Send + Debug {
+    /// Exports a batch of `ResourceLogs`.
```

Comment:
```suggestion
    /// Exports a batch of `LogData`.
```
nit: I believe the `ResourceLogs` is a concept of the OTLP protocol